src/js/actions/ui.js

```diff
@@ -412,7 +413,8 @@ define(function (require, exports) {
         var pathPromise = adapterUI.setSuppressTargetPaths(true);
 
         // Add additional shortcut CMD=, so that CMD+ and CMD= both work for zoom in.
-        var zoomInShortcutPromise = this.transfer(shortcuts.addShortcut, "=", { "command": true }, function () {
+        var zoomShortcutModifier = system.isMac ? { "command": true } : { "control": true };
+        var zoomInShortcutPromise = this.transfer(shortcuts.addShortcut, "=", zoomShortcutModifier, function () {
```

Comment:
These var declarations should either be comma separated, or new line separated.

Also a newline between 417 and 418 would make this code more readable.